src/pages/trade-types/cfds/_mind-when-trading.tsx

```diff
@@ -3,7 +3,7 @@ import styled from 'styled-components'
 import { graphql, useStaticQuery } from 'gatsby'
 import { SmallContainer } from '../components/_style'
 import { StyledLinkButton } from './_trading-cfd-increases'
-import { SectionContainer, Show } from 'components/containers'
+import { SectionContainer, Desktop, Mobile } from 'components/containers'
```

Comment:
Before changes around components Desktop and Mobile they are was from `components/containers/show` 

After changes the path is `components/containers/visibility`

A little confusing. Isn't better import them from `visibility.tsx`?

Reply:
@NikitaK-deriv that can be done, but with this common one import different components are being imported from different files in the components/containers folder or like in case of 'components/elements' an index file can be created which will import and export all the components in that folder.